include/nuttx/fs/fs.h

```diff
@@ -30,6 +30,8 @@
 #include <nuttx/config.h>
```

Comment:
#include <nuttx/config.h>

#include <nuttx/compiler.h>
#include <nuttx/fs/uio.h>

Reply:
do you mean to insert a blank line after config.h inclusion?

Reply:
Yes, it's the common style

Reply:
ok

Reply:
done